SPECS/cri-o/cri-o.spec

```diff
@@ -167,6 +149,21 @@ ln -sf service %{buildroot}%{_sbindir}/rccrio
 
 %fdupes %{buildroot}/%{_prefix}
 
+%post
+%systemd_post crio.service
+# This is the additional directory where cri-o is going to look up for CNI
+# plugins installed by DaemonSets running on Kubernetes (i.e. Cilium).
+mkdir -p /opt/cni/bin
+
+%post kubeadm-criconfig
+%fillup_only -n kubelet
+
+%preun
+%systemd_preun crio.service
+
+%postun
+%systemd_postun_with_restart crio.service
```

Comment:
Out of curiosity, how did you figure out it's `%systemd_postun_with_restart` and not `%systemd_postun`?

Reply:
openSUSE's `%systemd_postun_restart` macro is aliased to `%service_del_postun`. [Fedora uses](https://src.fedoraproject.org/rpms/cri-o/blob/rawhide/f/cri-o.spec) `%systemd_postun_with_restart` as well.